.bazelversion

```diff
@@ -1 +1 @@
-6.1.0
```

Comment:
Is this related?

Reply:
Oops, nope that's an unintended change. Sorry about that. Changed it because I was using a newer version of bazel:

> ERROR: The project you're trying to build requires Bazel 6.1.0 (specified in /Users/jtraglia/Projects/jtraglia/prysm/.bazelversion), but it wasn't found in /opt/homebrew/Cellar/bazel/6.1.1/libexec/bin.